transaction/examples/account/new.rtm

```diff
@@ -1,3 +1,4 @@
 CREATE_ACCOUNT_ADVANCED
-    Map<String, Tuple>();                  # Authority Rules
+    Enum<0u8>();
```

Comment:
In all of these `.rtm` files, can we instead use the string based values from `enums.rs` so people know what the type of this is?

Reply:
Sounds good, will update that.